http/src/main/java/com/redhat/lightblue/client/http/transport/HttpClient.java

```diff
@@ -0,0 +1,11 @@
+package com.redhat.lightblue.client.http.transport;
+
+import com.redhat.lightblue.client.request.LightblueRequest;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+public interface HttpClient extends Closeable {
+    // TODO: Do we need to able to examine headers or status code of response?
```

Comment:
Just returned String here since that's all that's technically need to keep 1:1 with existing behavior, but we may want to consider returning an `HttpResponse` object that has status code and other information from the request. This would probably be fine to be rolled into `LightblueResponse`, in which case this method would return that. Something to consider. For now this is simple and it works.